### ISSUE TITLE
自定义PASS Pattern中包含dropout算子，无法实现算子替换

### ISSUE THREAD

**Issue by ChengleiTian** [PaddlePaddle/PaddleCustomDevice#742]
HI， 我在开发自定义PASS，去实现推理时的算子融合，我发现如果模型中包含有dropout，PASS执行就会失败，无法成功替换为融合后的算子。下面是我简化后的代码，请问一下dropout有什么特殊之处吗？同样的代码如果换成Normalize就可以很好的工作。
# limitations under the License.

from __future__ import print_function, division

import os
import numpy as np
import unittest
import paddle

paddle.enable_static()

@paddle.incubate.passes.ir.RegisterPass
def generate_my_dropout():
    def pattern(x):
        return paddle.nn.functional.dropout(x)

    def replace(x):
        return paddle.incubate.passes.ir.PassDesc.OP.my_dropout(X=x)

    return pattern, replace

@paddle.jit.to_static(
    input_spec=[
        paddle.static.InputSpec([None, 32], "float32", "x"),
    ]
)
def func(x):
    return paddle.nn.functional.dropout(x)

MODLE_FILE = "./saved_dropout_model"

class TestCustomPass(unittest.TestCase):
    def setUp(self):
        for lib in os.listdir(os.getenv("CUSTOM_DEVICE_ROOT")):
            if lib.endswith(".so"):
                paddle.utils.cpp_extension.extension_utils.load_op_meta_info_and_register_op(
                    lib
                )
        paddle.jit.save(func, MODLE_FILE)

    def test_my_dropout(self):
        config = paddle.inference.Config()
        config.set_prog_file(MODLE_FILE + ".pdmodel")
        config.enable_memory_optim()
        config.enable_custom_device("intel_gpu")
        pass_builder = config.pass_builder()
        pass_builder.append_pass("generate_my_dropout")
        print(pass_builder.all_passes())
        predictor = paddle.inference.create_predictor(config)

        np_inputs = [
            np.random.randn(3, 32, 32).astype("float32"),
        ]
        input_names = predictor.get_input_names()
        for i, name in enumerate(input_names):
            input_tensor = predictor.get_input_handle(name)
            input_tensor.copy_from_cpu(np_inputs[i])

        predictor.run()
        results = []
        output_names = predictor.get_output_names()
        for i, name in enumerate(output_names):
            output_tensor = predictor.get_output_handle(name)
            output_data = output_tensor.copy_to_cpu()
            results.append(output_data)

if __name__ == "__main__":
    unittest.main()

**Comment by ronny1996** [comment]
```python
@paddle.incubate.passes.ir.RegisterPass
def generate_my_dropout():
    def pattern(x):
        dropout_op = paddle.incubate.passes.ir.PassDesc.OP.dropout
        dropout_op._inputs.pop('Seed')
        dropout_op._outputs.pop('Mask')
        return dropout_op(X=x)

    def replace(x):
        return x

    return pattern, replace
```

你好，pattern写成这样可以匹配上

**Comment by ChengleiTian** [comment]
> ```python
> @paddle.incubate.passes.ir.RegisterPass
> def generate_my_dropout():
>     def pattern(x):
>         dropout_op = paddle.incubate.passes.ir.PassDesc.OP.dropout
>         dropout_op._inputs.pop('Seed')
>         dropout_op._outputs.pop('Mask')
>         return dropout_op(X=x)
> 
>     def replace(x):
>         return x
> 
>     return pattern, replace
> ```
> 
> 你好，pattern写成这样可以匹配上

验证过了，可以匹配上，多谢！另外请问一下，如果被融合的算子有权重参数，如何提取出来传递给新算子使用？有没有相关的例子？
